cross/glib/patches/hi3535/001-fix-build-SYS_sched_getattr.patch

```diff
@@ -0,0 +1,20 @@
+--- meson.build-old	2021-01-09 14:43:01.994870325 +0000
```

Comment:
I would like to include such comment in the patch file (_the `-old => .orig` change is not relevant, but I had to change the existing line to make a suggestion_)

```suggestion
# see https://gitlab.gnome.org/GNOME/glib/-/issues/2274
# as this will not be maintained upstream we will have to maintain this patch
--- meson.build.orig	2021-01-09 14:43:01.994870325 +0000
```